sdk/python/kfp/dsl/component_factory.py

```diff
@@ -132,6 +133,14 @@ def make_pip_install_command(
 PIP_DISABLE_PIP_VERSION_CHECK=1 {pip_install_commands} && "$0" "$@"
 '''
 
+# Creates and activates a virtual environment in a temporary directory.
+# The environment inherits the system site packages.
+_use_venv_script_template = '''
+tmp=$(mktemp -d)
+python3 -m venv "$tmp/venv" --system-site-packages
+source "$tmp/venv/bin/activate"
```

Comment:
`source` is resulting in: 

```
sh: 12: source: not found
```

Since `source` is not available in shell (it's available in bash/zsh)

Can you replace this with `.` which is available in `sh`?

Reply:
Done. Thanks!